src/components/inline/Emphasis/index.js

```diff
@@ -0,0 +1,33 @@
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const Emphasis = (props) => {
+  const { children, ...otherProps } = props;
+
+  return <em {...otherProps}>{children}</em>;
+};
+
+export const styles = (props) => {
+  const { theme } = props;
+
+  return css`
+    background: ${theme.colors.text.emphasisBackground};
+    color: ${theme.colors.text.emphasis};
+    font-style: italic;
```

Comment:
I'm guessing this should all be theme-able _eventually_, but this is fine for now 😄